src/gba/gba.c

```diff
@@ -935,9 +935,9 @@ void GBAClearBreakpoint(struct GBA* gba, uint32_t address, enum ExecutionMode mo
 }
 
 #ifdef USE_DEBUGGERS
-static bool _setSoftwareBreakpoint(struct ARMDebugger* debugger, uint32_t address, enum ExecutionMode mode, uint32_t* opcode) {
+static ssize_t _setSoftwareBreakpoint(struct ARMDebugger* debugger, uint32_t address, enum ExecutionMode mode, uint32_t* opcode) {
 	GBASetBreakpoint((struct GBA*) debugger->cpu->master, &debugger->d.p->d, address, mode, opcode);
-	return true;
+	return 1;
```

Comment:
It's supposed to return the breakpoint ID, not a constant.

Reply:
I'm mistaken about this one. It seems the usage of this function shows that the API erroneously changed the return type, but still treated it as a bool.